collection-scripts/gather_dbginfo_z

```diff
@@ -0,0 +1,56 @@
+#!/bin/bash
+# dbginfo.sh is a script shipped with the s390-utils package on IBM-Z systems that collects system level information for debugging
+BASE_COLLECTION_PATH="must-gather"
+DBGINFO_PATH=${OUT:-"${BASE_COLLECTION_PATH}/node_dbginfo"}
+
+mkdir -p "${DBGINFO_PATH}"/
+
+function get_dbginfo_off_node {
+    local debugPod=""
+
+    #Get debug pod's name
+    debugPod=$(oc debug --to-namespace="default" node/"$1" -o jsonpath='{.metadata.name}')
+
+    #Start Debug pod, execute the dbginfo command to collect the system data and force it to stay up until removed in "default" namespace
+    oc debug --to-namespace="default" node/"$1" -- chroot /host /bin/bash -c 'mkdir -p /tmp/dbginfo && dbginfo.sh -d /tmp/dbginfo && sleep 300' > /dev/null 2>&1 &
+
+    #Mimic a normal oc call, i.e pause between two successive calls to allow pod to register
+    #Wait a minute for the dbginfo script to collect the information
+    sleep 60
+    oc wait -n "default" --for=condition=Ready pod/"$debugPod" --timeout=30s
+
+    if [ -z "$debugPod" ]
```

Comment:
I wonder about this check; what your checking for is that you define a pod name not that the pod is up(checked for by the wait command). 

Should we reconsider how this is done?

Reply:
Sleeping for some fixed amount of time can be problematic.  
Slow/resource-taxed environments will have to be the 'high water mark', every other environment will be hobbled by having to wait as long as the slowest cluster requires.  This isn't optimal.
If another way is possible, please consider it.

Reply:
i could remove the timeout and check if the condition is met.

Reply:
i am not sure how to avoid the sleep here or the timeout. we would need one or the other to give the pod some time.